Sources/Logging/Logging.swift

```diff
@@ -391,7 +413,11 @@ extension Logger {
     /// - parameters:
     ///     - label: An identifier for the creator of a `Logger`.
     public init(label: String) {
+        #if canImport(WASILibc)
```

Comment:
```suggestion
        #if os(WASI)
```

Reply:
Is `os(WASI)` available in all supported swift-log Swift versions, i.e. starting w/ Swift 5.0?